TestTools/StreamChatTestMockServer/Robots/ParticipantRobot.swift

```diff
@@ -310,20 +346,25 @@ public class ParticipantRobot {
     }
 
     @discardableResult
-    public func replyWithGiphyInThread(alsoSendInChannel: Bool = false) -> Self {
+    public func replyWithGiphyInThread(toLastMessage: Bool = true, alsoSendInChannel: Bool = false) -> Self {
         startTypingInThread()
         stopTypingInThread()
-
+        
+        let quotedMessage = toLastMessage ? server.lastMessageInThread : server.firstMessageInThread
+        let quotedMessageId = quotedMessage?[messageKey.id.rawValue] as? String
         let parentId = threadParentId ?? (server.lastMessage?[messageKey.id.rawValue] as? String)
         server.websocketMessage(
-            channelId: server.currentChannelId,
+//            channelId: server.currentChannelId,
```

Comment:
Should we delete this? Why is this commented out?

Reply:
Oops 🙈, fixed.